src/lint/lint.nim

```diff
@@ -43,8 +46,16 @@ Basic linting finished successfully:
 - Required track docs are present
 - Required shared exercise docs are present"""
   else:
-    echo """
+    echo &"""
 Configlet detected at least one problem.
 For more information on resolving the problems, please see the documentation:
-https://github.com/exercism/docs/blob/main/building/configlet/lint.md"""
+{url}"""
     quit(1)
+
+  if printedWarning:
+    echo ""
+    const msg = """
+      Configlet produced at least one warning.
+      These warnings will become errors in a future configlet release (at the end of October 2021).
```

Comment:
We'll have to change this if we add more warnings that won't become errors at the same time.